[2025-02-19T18:05:02.444+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-19T18:05:02.475+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: First_tag_example.first_task scheduled__2025-02-11T00:00:00+00:00 [queued]>
[2025-02-19T18:05:02.527+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: First_tag_example.first_task scheduled__2025-02-11T00:00:00+00:00 [queued]>
[2025-02-19T18:05:02.534+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-19T18:05:02.623+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-02-11 00:00:00+00:00
[2025-02-19T18:05:02.635+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'First_tag_example', 'first_task', 'scheduled__2025-02-11T00:00:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/first_dag_example.py', '--cfg-path', '/tmp/tmpfn8x752e']
[2025-02-19T18:05:02.633+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=259) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-19T18:05:02.642+0000] {standard_task_runner.py:105} INFO - Job 45: Subtask first_task
[2025-02-19T18:05:02.642+0000] {standard_task_runner.py:72} INFO - Started process 274 to run task
[2025-02-19T18:05:02.809+0000] {task_command.py:467} INFO - Running <TaskInstance: First_tag_example.first_task scheduled__2025-02-11T00:00:00+00:00 [running]> on host 4c8bb8eb4736
[2025-02-19T18:05:02.893+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='goncalo' AIRFLOW_CTX_DAG_ID='First_tag_example' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-11T00:00:00+00:00'
[2025-02-19T18:05:02.895+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-19T18:05:02.913+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-19T18:05:02.914+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hello world!"']
[2025-02-19T18:05:02.931+0000] {subprocess.py:99} INFO - Output:
[2025-02-19T18:05:02.936+0000] {subprocess.py:106} INFO - Hello world!
[2025-02-19T18:05:02.940+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-19T18:05:02.988+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-19T18:05:02.990+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=First_tag_example, task_id=first_task, run_id=scheduled__2025-02-11T00:00:00+00:00, execution_date=20250211T000000, start_date=20250219T180502, end_date=20250219T180502
[2025-02-19T18:05:03.061+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-19T18:05:03.074+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-19T18:05:03.076+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
